pyodide-build/pyodide_build/install_xbuildenv.py

```diff
@@ -20,32 +21,41 @@ def make_parser(parser: argparse.ArgumentParser) -> argparse.ArgumentParser:
         "Note: this is a private endpoint that should not be used outside of the Pyodide Makefile."
     )
     parser.add_argument("--download", action="store_true", help="Download xbuild env")
+    parser.add_argument("--url", help="URL to download xbuild env from", default=None)
     parser.add_argument("xbuildenv", type=str, nargs=1)
     return parser
```

Comment:
BTW, Do you know why we still have this argparse code and not using typer here? Do we still need it?

Reply:
I am trying to keep `pyodide_build` entrypoint and `pyodide` entrypoint compatible until we completely drop `pyodide_build` entrypoint. Maybe we should decide when we will drop `pyodide_build` entrypoint.